R/read.R

```diff
@@ -1,11 +1,10 @@
 orderly_read <- function(path, call = NULL) {
-  assert_file_exists_relative("orderly.R", name = "Orderly file",
-                              workdir = path, call = call)
-  orderly_read_r(file.path(path, "orderly.R"))
+  entrypoint_filename <- find_entrypoint_filename(path, basename(path))
```

Comment:
Almost every call here has this pattern. Consider refactoring so that you can do

```
find_entrypoint_filename(base)
```

(i.e., move the `basename` within `find_entrypoint_filename`) which will make this a little less noisy I think.

Reply:
done